src/frontend/Navigation/AppNavigator.tsx

```diff
@@ -33,6 +34,7 @@ import {Loading} from '../sharedComponents/Loading';
 export const AppNavigator = ({permissionAsked}: {permissionAsked: boolean}) => {
   const {formatMessage} = useIntl();
   const deviceName = useDeviceName();
+  usePrefetchLastKnownLocation();
```

Comment:
this is a relatively inexpensive call AND prefetching avoids and jumpy ui in the map!